server/src/service/authService.js

```
@@ -10,13 +9,11 @@ module.exports = {
         const client = new OAuth2Client(CLIENT_ID);
         var payload;
 
-        try {
-            payload = await token.verifyToken(client, tokenToVerify);
+        try { 
+            payload = await token.verifyToken(client, tokenToVerify); 
         }
         catch(ex) {
-            return new Promise(function(reject){
-                reject(constants.INVALID_TOKEN_RESPONSE);
-            });
```

Why aren't we returning promises here anymore?

---

For consistency of error handling within our app, we should be re-throwing the exception instead of returning a promise if there is an error. The existing code was broken, and so I either had to change the code here to throw an exception or we would have to handle the rejection in `server.js`. Since we are not handling rejections in any other methods within `server.js`, I decided to remove it so that we were consistent with the other methods.